web/src/main/java/no/nav/modiapersonoversikt/infrastructure/naudit/AuditResources.kt

```diff
@@ -75,6 +75,7 @@ class AuditResources {
             @JvmField
             val Kontrakter = AuditResource("person.kontrakter")
             val YtelserOgKontrakter = AuditResource("person.ytelserogkontrakter")
+            val siste14aVedtak = AuditResource("person.siste14aVedtak")
```

Comment:
Er det ikke bedre å bare sammenfalle alt under `14avedtak`? "Siste 14a vedtak" er litt for snevert imo.